packages/components/src/components/button-toggle/button-highlight.jsx

```diff
@@ -0,0 +1,28 @@
+import PropTypes from 'prop-types';
+import React     from 'react';
+
+const Highlight = ({ left, width }) => {
+    const border_radius_size = '4px';
+    const left_offset        = left;
+    const width_offset       = width;
```

Comment:
Since we removed the ternary expressions here, can we just use the `left` and `width` variables instead of making new `_offset` ones?